odoo/osv/expression.py

```diff
@@ -710,7 +710,7 @@ def push_result(query, params):
                 # Non-stored field should provide an implementation of search.
                 if not field.search:
                     # field does not support search!
-                    _logger.error("Non-stored field %s cannot be searched.", field)
+                    _logger.exception("Non-stored field %s cannot be searched.", field)
```

Comment:
That won't work.
`Logger.exception` only works in the context of an exception (in an `except` block).
What you want to use is the `stack_info` argument.
```suggestion
                    _logger.error("Non-stored field %s cannot be searched.", field, stack_info=True)
```

Reply:
Hello, thanks for your reply.

However, I have tried it locally and Sentry is able to retrieve the necessary information using exc_info=True

![image](https://user-images.githubusercontent.com/77889661/236406929-f3dfd1d6-59bb-4532-8243-33091460f09b.png)

(right: `logger.exception`, left: `logger.error`)

Reply:
And in the console/log file?

Reply:
In the console it doesnt change anything. Just the normal log without exception, like `logger.error`

So what you're suggesting is to have a stacktrace in the console in addition to Sentry, right?